claim_from_delivery/models/stock_picking.py

```diff
@@ -0,0 +1,22 @@
+# © 2020 Vanmoof BV (<https://www.vanmoof.com>)
+# © 2004-2009 Tiny SPRL (<http://tiny.be>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.model
+    def _claim_count_out(self):
+        """
+        Count number of calims ralted to stock pickings
+        :return: dict( picking.id: number of claims)
+        """
+        claim_cls = self.env['crm.claim']
+        for pick in self:
+            pick.claim_count_out = claim_cls.search_count(
+                [('model_ref_id', '=', 'stock.picking,{}'.format(pick.id))])
+
+    claim_count_out = fields.Integer(
+        compute="_claim_count_out", string="Claims")
```

Comment:
See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#models (and maybe it's good to reread this whole document): `the compute method pattern is _compute_<field_name>`